frontend/src/components/Pages/InspectionPage/ScheduleMissionDialogs.tsx

```diff
@@ -77,10 +77,12 @@ export const ScheduleMissionDialog = (props: IProps): JSX.Element => {
         if (!selectedRobot) return
 
         props.missions.forEach((mission) => {
-            BackendAPICaller.scheduleMissionDefinition(mission.id, selectedRobot.id).catch(() => {
+            BackendAPICaller.scheduleMissionDefinition(mission.id, selectedRobot.id).catch((e) => {
                 setAlert(
                     AlertType.RequestFail,
-                    <FailedRequestAlertContent message={`Failed to schedule mission ${mission.name}`} />
+                    <FailedRequestAlertContent
+                        message={TranslateText('Failed to schedule mission') + ` '${mission.name}'. ${e.message}`}
```

Comment:
There is translation on the message here but not other places (e.g. on line 117)